source/_cookbook/notify.mqtt.markdown

```diff
@@ -31,7 +31,7 @@ The same will work for automations.
 
 ### REST API
 
-Using the [REST API](https://developers.home-assistant.io/docs/en/external_api_rest.html) to send a message to a given topic.
+Using the [REST API](/developers/rest_api) to send a message to a given topic.
```

Comment:
This point to a redirect. Why not go directly for the target?

```suggestion
Using the [REST API](https://developers.home-assistant.io/docs/api/rest/ to send a message to a given topic.
```

Reply:
I was using the redirect in case that link was updated in the future and I thought that using the redirect was conventional (why else would it exist?)